src/common/OfflineNotice.js

```diff
@@ -30,15 +31,18 @@ class OfflineNotice extends PureComponent<Props> {
 
   render() {
     const { isOnline } = this.props;
-
-    if (isOnline) {
-      return null;
-    }
-
     return (
-      <View style={styles.block}>
-        <Label style={styles.text} text="No Internet connection" />
-      </View>
+      <AnimatedComponent
+        property="height"
+        useNativeDriver={false}
+        visible={!isOnline}
+        height={30}
+        style={styles.block}
+        animateOn="render"
```

Comment:
Consider all other uses of this component.
Will a "render" value make sense for all the rest? If so, we should just use that and not make it optional?

Reply:
This prop is still here at this call site; I think you meant to remove it?

Reply:
Yup, thanks!